beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestWithBCDUpdate.java

```diff
@@ -793,4 +787,72 @@ public void testWithBCDOnOperations() throws Exception {
 
 
     }
+
+    @Test(groups = "slow")
+    public void testBCDChangeForConsumableInArrearPlan() throws Exception {
+        // We take april as it has 30 days (easier to play with BCD)
+        // Set clock to the initial start date - we implicitly assume here that the account timezone is UTC
+        clock.setDay(new LocalDate(2012, 4, 1));
```

Comment:
Nit: Potentially we could update the start date - e.g `2016` should behave like `2012`